Libraries/D3D12RaytracingFallback/src/FallbackLayerUnitTests/fallbacklayerunittests.cpp

```diff
@@ -2769,6 +2770,78 @@ void AllocateUAVBuffer(ID3D12Device &d3d12device, UINT64 bufferSize, ID3D12Resou
             }
         }
 
+        TEST_METHOD(LoadTriangles)
+        {
+            ID3D12Device &d3d12Device = m_d3d12Context.GetDevice();
+            LoadPrimitivesPass loadPrimitivesPass(&d3d12Device, 1);
+
+            CComPtr<ID3D12GraphicsCommandList> pCommandList;
+            m_d3d12Context.GetGraphicsCommandList(&pCommandList);
+
+            std::vector<Triangle> triangles;
+            triangles.push_back({ -2.0f,  1.0f, -2.0f,  1.0f,  1.0f, -2.0f,  1.0f,  1.0f,  1.0f });
```

Comment:
Bit of a nitpick but can you move these to be initialized by a for-loop? You can initialize the contents of the triangles by using the loop counter. This has the nice side-effect where updating this test to be more of a stress-test just requires updating some const defining the number of triangles to test rather than a mass of copy-paste